modules/apps/frontend-js/frontend-js-react-web/.npmbundlerrc

```diff
@@ -59,6 +59,9 @@
 		],
 		"whatwg-fetch": true
 	},
+	"ignore": [
+		"webpack/*"
```

Comment:
This feels odd... are we cleaning up this folder? Do we need to `.gitignore` it? Can't we use something already ignored for our build artifacts?

Reply:
No need to gitignore, as it is created under the `output` directory so it's already ignored. We just need to make sure that bundler 2 doesn't touch it, because otherwise it will wrap those files inside a `Liferay.Loader.define()` call, which will break anything.

Then, when the whole portal is building under webpack 5, we may remove bundler 2 and, thus, this file entirely.

Reply:
Let's add this to the default preset or configuration, then

Reply:
done -> https://github.com/izaera/liferay-frontend-projects/commits/issue-250